src/dataPoint.js

```diff
@@ -2,123 +2,132 @@
  * Copyright (C) 2023 Radix IoT LLC. All rights reserved.
  */
 
-const uuid = require('uuid/v4');
+const { v4: uuid } = require('uuid');
 
 function dataPointFactory(client) {
     const MangoObject = client.MangoObject;
 
     return class DataPoint extends MangoObject {
-        
         static get defaultProperties() {
             const xid = uuid();
             return {
-                xid : xid,
+                xid: xid,
                 name: xid + ' Name',
-                enabled : false,
-                deviceName : xid + ' Name',
-                dataSourceXid : null,
-                useIntegralUnit : false,
-                useRenderedUnit : false,
-                readPermission : '',
-                setPermission : '',
-                chartColour : '',
-                rollup : 'NONE',
-                plotType : 'STEP',
-                loggingProperties : {
+                enabled: false,
+                deviceName: xid + ' Name',
+                dataSourceXid: null,
+                useIntegralUnit: false,
+                useRenderedUnit: false,
+                readPermission: '',
+                setPermission: '',
+                chartColour: '',
+                rollup: 'NONE',
+                plotType: 'STEP',
+                loggingProperties: {
                     loggingType: 'ALL',
                     discardExtremeValues: false,
                     discardLowLimit: null,
                     discardHighLimit: null,
-                    cacheSize: 1
+                    cacheSize: 1,
```

Comment:
Why add the trailing comma to the last property? I cannot remember the intricacies around this, but I don't think these are strictly equivalent. Given this is running only in Node it may not matter though.